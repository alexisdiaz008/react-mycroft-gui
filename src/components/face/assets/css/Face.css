.face {
	width: 100%;
	position: fixed;
	padding: 50% 0;
}
