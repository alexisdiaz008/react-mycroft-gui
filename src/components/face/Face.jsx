import React, { Component, useState } from 'react';
import restingFace from './assets/images/black-hole.gif';
import './assets/css/Face.css';

const Face = () => {
  const [resting, setResting] = useState(true);
  return (
      <img
    src={restingFace}
      className="face row"
      alt="logo"
     />
  )
}

export default Face;
