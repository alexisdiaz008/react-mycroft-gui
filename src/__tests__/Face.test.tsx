import * as React from 'react';
import { shallow } from 'enzyme';
import Face from '../components/face/Face';

const wrapper = shallow(<Face />);

describe('Face component', () => {
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
