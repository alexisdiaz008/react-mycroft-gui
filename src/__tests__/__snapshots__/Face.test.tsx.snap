// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Face component renders correctly 1`] = `ShallowWrapper {}`;
